lib/tools/android-manifest.js

```diff
@@ -98,7 +98,8 @@ manifestMethods.targetSdkVersionFromManifest = async function (localApk) {
     let targetSdkVersion = new RegExp(/targetSdkVersion:'([^']+)'/g).exec(stdout);
     return parseInt(targetSdkVersion[1], 10);
   } catch (e) {
-    log.errorAndThrow(`targetSdkVersionFromManifest failed. Original error: ${e.message}`);
+    log.warn(`Ran into problem getting target SDK; ignoring: ${e.message}`);
```

Comment:
This isn't exactly what I had in mind.

What you've done here is to make `targetSdkVersionFromManifest` emit a warning instead of throwing an error.

But I think this function _should_ throw an error if something fails.

It's in the _calling_ function below that I think we should explicitly ignore the error. In other words, `targetSdkVersionFromManifest` is a library function, and as such it should indeed throw any errors it encounters. But based on how we are using it in `install` here, we may or may not care about the error. That in my mind is a more flexible structure.

What do you think?